.github/workflows/fetch-enisa-euvd-list.yml

```diff
@@ -0,0 +1,101 @@
+name: Fetch EUVD List
+
+on:
+  workflow_call:
+  workflow_dispatch:
+    inputs:
+      fetch_signle:
+        description: "Fetch EUVD list with signle thread"
+        required: true
+        default: false
+        type: boolean
+
+jobs:
+  check:
+    name: Decide whether to fetch signle
+    runs-on: ubuntu-latest
+    outputs:
+      do_fetch_signle: ${{ steps.check.outputs.fetch_signle }}
+    steps:
+      - name: Check if fetch_signle
+        id: check
+        run: |
+          if [[ "${{ github.event_name }}" == "workflow_dispatch" ]]; then
+            echo "fetch_signle=${{ inputs.fetch_signle }}" >> $GITHUB_OUTPUT
+            exit 0
+          fi
+
+          if [[ $(( ${{ github.run_number }} % 15 )) -eq 0 ]]; then
+            echo "fetch_signle=true" >> $GITHUB_OUTPUT
+          fi
```

Comment:
I couldn't come up with the idea why single threaded execution was adopted here.
For correctness? For reliability? If there is some comment that explains WHY, it would be appreciated.